pkg/exchange/binance/parse.go

```diff
@@ -462,6 +467,60 @@ func parseDepthEvent(val *fastjson.Value) (*DepthEvent, error) {
 	return depth, err
 }
 
+type MarketTradeEvent struct {
+	EventBase
+	Symbol   string           `json:"s"`
+	Quantity fixedpoint.Value `json:"q"`
+	Price    fixedpoint.Value `json:"p"`
+
+	BuyerOrderId  int64 `json:"b"`
+	SellerOrderId int64 `json:"a"`
+
+	OrderTradeTime int64 `json:"T"`
+	TradeId        int64 `json:"t"`
+
+	IsMaker bool `json:"m"`
+}
+
+/*
+
+market trade
+
+{
+  "e": "trade",     // Event type
+  "E": 123456789,   // Event time
+  "s": "BNBBTC",    // Symbol
+  "t": 12345,       // Trade ID
+  "p": "0.001",     // Price
+  "q": "100",       // Quantity
+  "b": 88,          // Buyer order ID
+  "a": 50,          // Seller order ID
+  "T": 123456785,   // Trade time
+  "m": true,        // Is the buyer the market maker?
+  "M": true         // Ignore
+}
+
+*/
+
+func (e *MarketTradeEvent) Trade() types.Trade {
+	tt := time.Unix(0, e.Time*int64(time.Millisecond))
+	return types.Trade{
+		ID:            uint64(e.TradeId),
+		Exchange:      types.ExchangeBinance,
+		Symbol:        e.Symbol,
+		OrderID:       uint64(e.BuyerOrderId),
+		Side:          types.SideTypeBoth,
```

Comment:
Both is used for self trade type

Reply:
shouldn't self trade be `SideTypeSelf`?  
for this case, we are not either buyer or seller, and we don't take our own order.
so should be `SideTypeBoth`.